### ISSUE TITLE
Handle plurals

### ISSUE THREAD

**Issue by omegabytes** [omegabytes/askTheDM#108]
*Current output:* "10 gold piece"
*Optimal output:* "10 gold" _or_ "10 gold pieces"

**Comment by Eldres** [comment]
has this been resolved?

**Comment by omegabytes** [comment]
No, we need to add both single and plural forms of words to the dictionary. For example: candle, candles. Probably the  best way to do this is by adding a plural/singular key/val pair to the item object in the json, like:
"candle" : {
    "plural" : "candles",
   ...
}
